ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -400,6 +400,32 @@ Verify Event Log Is Accessible While Starting A Workbench
     [Teardown]    Clean Project From Workbench Resources    workbench_title=${WORKBENCH_6_TITLE}
     ...    project_title=${PRJ_TITLE}
 
+Verify User Can Cancel Workbench Start From Event Log
+    [Tags]    Sanity    Tier1    ODS-1975
+    [Documentation]    Verify user can cancel workbench start from event log
+
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}  workbench_description=${WORKBENCH_DESCRIPTION}
+    ...        prj_title=${PRJ_TITLE}    image_name=${NB_IMAGE}   deployment_size=Small
+    ...        storage=Persistent  pv_name=${NONE}  pv_existent=${NONE}
+    ...        pv_description=${NONE}  pv_size=${NONE}
+    ...        press_cancel=${FALSE}    envs=${NONE}
+
+    Workbench Status Should Be    workbench_title=${WORKBENCH_TITLE}  status=${WORKBENCH_STATUS_STARTING}
+
+    Open Notebook Event Log    workbench_title=${WORKBENCH_TITLE}
+    Page Should Contain Event Log
+    Cancel Workbench Startup From Event Log
+    Wait Until Workbench Is Stopped    workbench_title=${WORKBENCH_TITLE}
+
+    Workbench Status Should Be    workbench_title=${WORKBENCH_TITLE}  status=${WORKBENCH_STATUS_STOPPED}
```

Comment:
it's redundant with the previous line. `Wait Until Workbench Is Stopped` internally uses `Workbench Status Should Be`

Reply:
`Wait Until Workbench Is Stopped` does not have `Should` in it and it does not feel like i am making an assertion. And if I wanted to run `Workbench Status Should Be` repeatedly, then I'd have to set a check frequency and timeout which feels like extraneous detail when reading the test.

I'll use `Wait Until Workbench Is Stopped` only and remove `Workbench Status Should Be`.

Reply:
> does not have Should in it and it does not feel like i am making an assertion. 


yep, makes sense but it is technically doing the same check, it would be a line of code we could spare. But if you think the benifits of keeping it are important enough, feel free to do it :) 


> [...] which feels like extraneous detail when reading the test.


Maybe this logic can be applied to the following lines as well:

```
Wait Until Keyword Succeeds  5 min  5s
    ...    Run Keyword And Expect Error  EQUALS:ResourceOperationFailed: Get failed\nReason: Not Found
    ...    Get Workbench Pod    project_title=${PRJ_TITLE}  workbench_title=${WORKBENCH_TITLE}
```

Reply:
> Maybe this logic can be applied to the following lines as well:

I guess so. I did not have any bright ideas about what to do instead. And I still don't.

Reply:
You could create a separate keyword